### DISCUSSION TITLE
What is Partial Embed? When is it available?

### DISCUSSION THREAD

**Discussion by taowang1993** [infiniflow/ragflow#2425] [General]
<img width="236" alt="image" src="https://github.com/user-attachments/assets/12f33042-ba5c-4a26-b792-ffcc28f8b25b">

**Comment by KevinHuSh** [comment]
Where is that come from?
RAGFlow does not have this  feature, doesn't it?

**Reply by taowang1993** [reply]
it does
<img width="881" alt="image" src="https://github.com/user-attachments/assets/78fd1d8d-109d-47bf-91da-cc989fa78bfd">

**Reply by taowang1993** [reply]
The iframe embedding approach will expose url. 
Bad actors can use my chatbot or agent without paying.
They can just inspect the code and copy the url, then consume my tokens.

**Reply by KevinHuSh** [reply]
Got it. 
We're working it.
